content/docs/user-guide/components/reference/gradients/fastnoise-gradient.md

```diff
@@ -6,13 +6,33 @@ description: Use the FastNoise Gradient component to generate a gradient with th
 
 Add the **FastNoise Gradient** component to use one of the noise generation algorithms from the [FastNoise](https://github.com/Auburn/FastNoiseLite) library to generate a gradient.  The noise generation algorithm is set in the component's **Noise Type** property.
 
+## Noise types
+
+| Noise Type | Description | Example Gradient |
+| - | - | - |
+| `Value` | Generates interpolated values directly from XYZ-coordinates to produce a zoomed-out version of `White Noise`. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-value.png) |
+| `Value Fractal` | Results from the `Value` algorithm are run through a fractal function. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-value-fractal.png) |
+| `Perlin` | Generates values from the Perlin noise algorithm, a noise algorithm where visual features are all of similar size. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-perlin.png) |
+| `Perlin Fractal` | Results from the `Perlin` noise algorithm are run through a fractal function. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-perlin-fractal.png) |
+| `Simplex` | Generates values from the Simplex noise algorithm, a variation of Perlin noise with fewer directional artifacts. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-simplex.png) |
+| `Simplex Fractal` | Results from the `Simplex` noise algorithm are run through a fractal function. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-simplex-fractal.png) |
+| `Cellular` | Generates values from a cellular noise algorithm, which assigns values based on randomly distributed _feature points_; each world position is assigned the value of the closest feature point. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-cellular.png) |
+| `White Noise` |	Generates values from XYZ-coordinates, which produce extremely different values for adjacent samples. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-white-noise.png) |
+| `Cubic` | Generates values directly from XYZ-coordinates that are then run through cubic interpolation with neighboring values. This is a zoomed-out version of `White Noise` run through cubic interpolation. The results are similar to `Perlin` noise, but with fewer directional artifacts and with a higher occurrence of extreme values. | ![FastNoise value noise type example](/images/user-guide/components/reference/gradients/fastnoise-cubic.png) |
```

Comment:
Same feeback as `Value` - this seems to be based in world units. If that's the case, we should try to explain that rather than refer to is as zoomed-out.